sdk/devcenter/Azure.Developer.DevCenter/tests/Samples/Sample_CreateDeleteDevBoxAsync.cs

```diff
@@ -16,11 +14,14 @@ public partial class DevCenterSamples: SamplesBase<DevCenterClientTestEnvironmen
         public async Task CreateDeleteDevBoxAsync(Uri endpoint)
         {
             // Create and delete a user devbox
-            #region Snippet:Azure_DevCenter_GetProjects_Scenario
+            #region Snippet:Azure_DevCenter_CreateClient_Scenario
```

Comment:
I see you have both generated samples and not generated samples. Curious what extra samples did you need that generated samples did not cover for you ?

Reply:
The manually created samples show more of an end-to-end "hero" scenario for both devbox and dev. environments. Whereas the generated samples target individual methods.